### ISSUE TITLE
[Feature Request]: Update toolkit to use Segoe UI as the default font

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Standard-Toolkit#855]
Very simple, update toolkit to use Segoe UI or Segoe UI Variable if >= Windows 11 as the default font

**Comment by Smurf-IV** [comment]
No.. I think it should be the default since it was supposed to be the default for Windows 7 onwards..

But, I still prefer Tahoma, but that might be because I've been doing this too long ;-)

Also this will also need to be placed into the Extended toolkit as well.
AND, it should be a theme thing !
